### DISCUSSION TITLE
"failed to parse date field" Logstash Error Messages

### DISCUSSION THREAD

**Discussion by imaohw1337** [Security-Onion-Solutions/securityonion#4777] [Unsupported Versions] [answered]
After update to 2.3.60 "failed to parse date field" messages are flooding "/opt/so/log/logstash/logstash.log" on search node. Logs are coming in at near 90gb daily since the update.

response=>{"index"=>{"_index"=>"<so-%{[event][module]}-2021.07.12-{2021.07.12||/d{yyyy.MM.dd|UTC}}>", "_type"=>"_doc", "_id"=>nil, "status"=>400, "error"=>{"type"=>"parse_exception", "reason"=>"failed to parse date field [[event][module]] with format [uuuu.MM.dd]: [failed to parse date field [[event][module]] with format [uuuu.MM.dd]]", "caused_by"=>{"type"=>"illegal_argument_exception", "reason"=>"failed to parse date field [[event][module]] with format [uuuu.MM.dd]", "caused_by"=>{"type"=>"date_time_parse_exception", "reason"=>"Text '[event][module]' could not be parsed at index 0"}}}}}}

All of these logs have ""type"=>"filebeat", "version"=>"7.13.2", "hostname"=>"so-filebeat"" in them.

Seems like I'm getting this error from most logs coming in (syslog, ossec, beats). I've got my normal amount of events coming into Elasticsearch that appear in Kibana. The amount of logs being generated is concerning as it filled up my system-root over the weekend.

If anyone has encountered this, please send tips.

Thanks.

**Comment by mtnsec** [comment]
I have the same errors in my logs since 2.3.60, and as far as I can tell I'm not missing any events in Elasticsearch either.

I have no idea what the cause or solution is - just wanted to chime in and let you know you're not alone. :)

**Comment by dougburks** [comment, marked as answer]
We just released a hotfix that should address this:
https://blog.securityonion.net/2021/07/security-onion-2360-filebeat-pipeline.html

**Reply by paintedrealms** [reply]
running 3.2.70 but still failing to parse udmpro syslogs

**Reply by dougburks** [reply]
@paintedrealms Per https://github.com/Security-Onion-Solutions/securityonion/discussions/1720, please start a new discussion instead of replying to somebody else's discussion.
